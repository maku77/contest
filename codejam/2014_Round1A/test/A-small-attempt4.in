100
1 2
11
01
4 8
01101011 10100010 01011101 10010100
10100011 01011100 10010101 01101010
4 8
01110001 01010011 11011010 01010100
00010000 00110101 00010111 10011110
10 10
1101100111 0111111010 1000011111 1010000101 0100100011 0110011000 1000000100 0011111000 0101101000 1011010111
1010111101 0100100001 1011011111 1001001101 1000000110 0111110010 1111011101 0001000010 0100111010 0110100000
6 9
011110111 101111100 001101110 010111000 000111100 101011111
111110111 001011111 000010000 011000110 111010100 010010100
2 9
111100011 001110111
011110111 101100011
2 7
1011011 1111011
1100101 1000101
8 9
101000101 010110110 101001101 011111110 011110110 100000101 101000001 011110010
011110010 100000101 011111110 011110110 010110110 101000001 101001101 101000101
4 4
0000 1010 1000 0010
1001 1011 0001 0011
6 6
111100 111101 101100 110010 100010 101101
101011 111011 110100 111010 100100 101010
3 9
001110101 001110100 010000101
011001000 111011010 001110100
10 10
1110110000 1001111000 1011001001 0100011010 1011100110 1101000011 0010100001 1100111000 1101010010 1101110010
1110011111 0110111110 1010100101 0011011001 1111010000 1100100010 0111111010 1101110011 1101100000 0101101011
10 10
0000000111 1011010001 0101011010 0100111001 0100110101 0010000000 1100111100 1011000110 1011101111 1111101010
1111010111 0100111111 1000000100 1111111110 0001100010 0000000001 0000001101 0110111000 1011010010 1111101001
5 6
011001 011000 100010 111011 010111
110100 000011 000010 100101 101111
3 5
01110 11011 11010
01101 01100 11001
10 10
0010011000 1101111111 0111100010 1100100100 0111001011 0001100010 0000010111 1010010100 1100011001 1011101101
0100000010 1111111000 1000001100 0011011110 1000110001 0110001101 1110000001 0011110111 1001101010 0101110111
10 8
11100110 00100001 10111001 11011001 01000100 00010111 10100101 11111001 01010010 10000010
11011110 10110100 10011001 11000011 01011010 00000111 00110111 00100101 10001100 11101000
10 10
0101111011 1110001011 1111101001 0011100101 0000011100 0000110010 1010010111 1011001000 0101110100 1100000110
1011111101 0101101010 1110111110 0101000100 1010011111 0110010011 0000000010 1001110000 1111100001 0000001101
10 10
1001111011 0111000100 1001110010 0101000100 1010001100 0110111011 0110101111 0000000111 1010110000 1101011001
1011000011 1000000001 0010110110 1000111101 0111110101 0100011110 1011001010 0101110101 0100001010 1111101000
10 10
1100000010 1111001100 1111100101 0001001111 0101001011 0010111011 0010110000 1000010100 0011111110 1100110001
1000011100 1000110101 0101001001 0010110010 0110110110 0101000010 1011001000 0100000111 1111101101 1011111011
8 9
000011101 100101100 010000011 110010010 000011011 000110110 111110001 110011001
001100001 001101010 000000010 111000101 011011111 111101110 101110000 111101000
7 7
0010001 0111001 0110001 1010000 0010011 0111100 1011011
0001011 0000110 0100100 0001110 0100110 1100111 1101100
10 10
1100101001 1111111011 0010110000 0000000101 0101010000 1000011110 1000000100 1011100111 0111011011 0111101110
1100110001 1001100100 1011011010 0100101010 0111010011 0000011101 0100110000 1110000100 1011101111 0011001111
7 10
0010100111 0001011000 0000010111 1100000011 1000010111 1011110000 1110101001
1101001000 0111000001 0111011110 0001001011 0010001101 1100110101 0111101011
3 7
0110000 0010010 0000101
0110010 0010000 0100101
7 9
011010000 110011111 110101101 100111100 011011011 111001111 110011110
010111001 101000001 100111110 101101110 001110011 100100111 000010101
4 3
100 110 111 010
100 111 001 101
6 4
1110 0000 1101 0111 0100 0010
0101 0011 1001 0000 0111 1010
4 6
100001 111001 011101 000101
011110 111010 000110 100010
1 8
11001011
11111111
9 9
000010111 000011101 100001101 011100100 011001111 011010011 001111101 111011000 100111111
110010000 101000100 111010110 010001101 100111011 010111000 100011010 110000010 000100000
2 2
00 01
11 10
10 10
1001101100 1101001010 1111110111 0000110000 1010010101 0100010011 0100111000 0011000010 1110101111 0011001101
0001000100 0010111101 1100111011 0111011111 0110000111 1011101010 0101100010 1100010000 1000011000 1011100101
3 3
001 111 010
010 001 100
8 10
1000100000 0111010101 1000101111 0111111111 0111110000 0111111101 1000101101 1000000101
0111110110 0111010011 1000100110 1000000011 1000001100 0111011110 0111011100 1000001110
10 10
0111001011 1010011011 0000001000 1100000001 0100101010 0101111110 1111100101 0011110101 1001010110 1010110100
1111000010 0000100111 0110111101 1001001001 1110010110 1101110111 1010110100 0011101010 0101011001 0000001000
4 8
10110101 00011101 11000110 00011011
00100101 11111000 10001011 00100011
4 6
110000 100100 000110 010010
010001 100111 000101 110011
4 9
100001100 111001100 101011100 110100000
010101110 011011110 011101010 101010101
2 3
000 100
010 110
4 8
10000000 10110001 00011100 00100111
01001011 00100110 00001111 11111010
10 10
0011011011 1111101000 1100001110 0000010110 1010000001 1001110101 0111110100 0000101010 1110001111 0101110001
1101101110 0011101111 1110011010 0100001101 0100110001 1000010101 1011110011 0001101010 1010010100 0111000000
8 6
101101 101011 101110 111011 001111 111110 111101 011111
001001 101101 111101 011111 011001 011100 001111 001100
5 6
000010 001110 101001 010011 011101
010110 110011 000011 100011 000001
10 10
1010101011 1111001100 1100010101 1100001001 0001110100 0001100000 0010101011 0011111010 1110010110 0101010111
0100010011 1111010010 1000101110 1001111111 0000101110 1011110001 0110010000 1011100101 0110001100 0101001001
8 9
001011111 111001001 011011010 000101000 011100001 101100001 010011111 100011010
000011110 000001001 111001011 100111100 110010001 100000010 010100000 011111000
4 6
101111 100011 110010 111110
101111 100011 110010 111110
4 9
111010010 000111001 100110011 011011000
010011100 001111101 110010110 101110111
2 8
00011011 11010111
00101001 11100101
8 8
10101001 11110000 01110100 10001100 01111001 01000100 01101111 10010110
10001001 10010011 10110110 01011011 01100110 01101011 11101111 01110000
2 9
010101101 100101010
100011010 010011101
10 10
1011011011 1001100110 0111000001 0100001101 1000011111 1100100010 0010011001 1011100110 0100111000 0111110100
1011000000 0000010110 1110101101 0111101111 0111010010 0101010010 0100101011 1000001100 1011110101 1000111001
10 9
110110111 101011000 001010010 010110111 101010010 110111101 110100111 010111101 101000010 001011000
001000010 101000010 010100111 001001000 101010010 110110111 010101101 110101101 110100111 101001000
10 10
0101101001 0000011001 1010100111 0110001101 0101010010 1111101110 0000110000 1011000010 1110111110 1001010101
0011101001 1111000011 1001111110 1001010111 1100011100 1100100111 0000011011 0110100000 0111110000 0010001100
1 3
100
111
4 5
11000 01100 00011 00010
11110 10000 10001 01010
2 4
1101 1010
0101 0000
4 8
01000001 11010001 10000111 01100000
10111100 11101100 01011101 00110000
4 2
01 11 10 00
00 11 01 10
10 10
0001011101 1110101100 1110110001 1100100111 0011010000 1100011010 0111110110 0001100111 1011001001 0000001010
1000010000 0100111101 0011010011 0110101011 1011001010 0110110110 1001000111 1111101100 0100000000 1001111101
10 10
0110100100 1101010010 1001001111 0111110010 1011011010 0100110101 0101101110 1000101001 0010001001 1010010101
0000101101 0011011110 1100110001 1101101010 0010010001 1111110110 0101010110 1010001101 0001001011 1110100000
8 6
011000 001001 110100 000111 101000 111010 110001 010101
110010 111100 000001 100011 101110 001111 001010 010011
10 10
0101111010 1101111010 0001001111 1010000011 1101100001 0010110100 0110111100 0010001001 1001010100 1110000111
1000000101 0111001011 1111111100 1000011110 0100110000 0000000101 1011111000 1100101011 0111110110 0011000011
10 10
1010000100 0001110000 0000101111 0100110111 1111001010 1011000001 1010101100 0101011011 1111010100 0100111011
1001110111 0110101000 0011011100 0111000100 1100111001 0111001000 0010000011 1000110010 1100100111 1001011111
3 3
110 101 010
111 000 100
10 10
0110100100 1011001111 1001000111 1001100000 0111111111 1010110101 0011011001 0100001010 1100011000 0100110010
0000100000 1100110001 0101011101 1111100100 1010011100 1111000011 1101001011 0010110110 0010001110 0001111011
4 4
1110 0010 0000 1100
0110 1010 0100 1000
4 7
0001111 0011001 1100011 1101110
1110100 0010101 0000011 1111001
10 7
0101000 0100001 1110011 1100101 0010100 0010110 0000100 1000000 0111101 0011101
0100110 0000101 1011010 0110111 0110011 1011111 1101101 0000111 1100010 0101110
10 10
0010011001 0111101101 1010000100 1001110110 1001010001 0101001111 1101100010 0000101010 0110010101 1110111010
1001111110 1110101000 0010010001 0101011101 0010110110 1101110010 0110000101 0001100011 1100001010 1011001101
8 8
01011000 00010010 11011111 00110000 10100100 11000110 00001000 01001100
01010000 11000100 00111000 01101000 01110010 00101100 10111111 10100110
9 9
001011100 001000011 101000100 010111010 010110011 100001001 011100010 001101001 011010011
010101111 000010100 111110101 011101111 010110011 000000011 010010111 010010010 100010111
10 10
1101110010 0101011111 0000001010 1111000100 0011110100 0000100001 0100001011 1011100101 1010011010 1110111101
1011010100 0000010000 0011000101 0101101110 1111110101 1110001100 1010101011 0100111010 0000111011 1101000011
3 6
010010 001111 010011
100101 111000 100100
10 10
0100011001 1000011110 1111001001 1011010100 0110100111 1111101101 0001110010 1010111010 0000000000 0101100111
1100010110 1001111101 0111100010 1101101000 1000001111 0000010001 1110101000 0011011011 0111000110 0010110101
10 7
1001011 1011011 1111010 1001100 1101010 1110101 1000001 1011100 1010001 1100101
1100000 1011001 1001110 1010100 1111111 1000100 1110000 1011110 1101111 1001001
2 7
1111110 0110000
0100001 1101111
9 10
1101100001 1111100001 0010000001 0001010100 0101111111 0001010111 1001011111 0011110000 0100000100
1101001110 0111100011 0110100011 0111010001 0011110011 1110111101 1111110101 1100011011 1001001100
10 10
0000100000 1010001100 0001011111 1111110001 1001000110 0100001111 1100100011 0110010101 0011111000 1111111010
1000100011 0000111010 1111101001 0101011111 1010000100 1001011100 1101110011 0011110000 0110001101 0110000110
10 10
0000111101 1100111010 0111100110 1000110111 0011000001 1011001110 0100101000 1111010101 0001000011 1110011000
0111001010 1001011100 1100110111 0100100101 0110000111 1011011110 1000100010 0011010001 1111111001 0000101000
10 9
111010011 001111100 111111110 110010001 111111100 011111100 001010001 110111100 111010001 011010001
010101100 111000001 011101110 010000001 101101100 011101100 011000011 101000001 111101100 011000001
10 10
1111110101 1011001101 1100111010 1111011010 0011010010 1000100001 0000110100 0100100111 0000001100 0111001011
0010001111 1001111110 0101001000 0110010110 0101110000 1010011110 0001100011 1101100101 1010110001 1110001001
10 10
1000110110 1111101100 0101110011 1110001001 1000010001 0101111101 1110101110 0001000011 0010000100 0011011010
1101010000 0001100110 1100110101 1010001010 0011111111 1010101101 1100010010 0111000001 0111001111 0000111000
1 7
1111110
0010001
1 2
01
10
4 7
1110100 0100100 0110001 0010101
0101110 0000010 1011110 0010000
10 10
0110011111 1001000101 0101100001 0000110010 0101111100 1010111011 1011001101 0101010010 1110000110 1010101000
1001101011 0101001111 0110100010 0110110001 1100111000 0010001100 0111000111 1010010101 1001110110 1001011000
1 10
1101110011
1010101010
4 9
111111111 011111010 001001010 101001111
011011000 001101000 101101101 111011101
2 4
1101 1111
0110 0100
9 9
110010001 000010001 101000110 010110110 111111111 101000111 101001110 011100010 110110011
001000101 110110101 010110000 001001101 001000100 111100001 100010010 011111100 010010010
3 7
0111011 1000011 1101100
1111001 0101110 0000001
10 10
1110101101 1101101100 0010001111 1110010001 1101100110 1010111001 0011010110 0000111010 0001010001 0101000010
1101110100 0101000010 1010111001 1111000001 0001101010 1110101010 0001010110 1100101101 0010010111 0010011101
7 7
1111010 1011100 1101111 1010101 1000111 0010110 1001111
0000100 1000101 0001010 1011110 0111101 0010011 1011100
9 10
0101111111 1111110101 0111111001 0001000111 0011010110 0111111101 0101011011 0010011111 0111110111
1101101110 0000101110 0100110111 1001111000 1001101000 0001000110 0010111101 0101111001 1110111111
6 4
0111 0000 0101 0010 1000 1101
1111 0010 1010 0111 1000 1101
10 10
0111001101 0100010100 0011111011 1001110110 1100010111 0011100101 0000100010 1111011000 1110101001 1000001010
1110011101 1001101100 0011111111 0100100000 1101000100 0110011110 1010101011 0010000011 0101010001 1001110010
1 2
01
00
5 4
0111 1000 1100 1111 0000
0100 1111 0000 0111 1000
10 10
0001010110 1010111001 0010110010 0001101111 0101101000 0100000010 1101010110 1111001101 1010001101 1110110001
0111111110 0000010001 1111000000 1000011010 1011111110 0100011001 0101100101 1110101010 1011000111 0000100101
