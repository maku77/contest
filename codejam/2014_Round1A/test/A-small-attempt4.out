Case #1: 1
Case #2: 1
Case #3: 2
Case #4: 5
Case #5: 3
Case #6: 1
Case #7: 4
Case #8: 0
Case #9: 1
Case #10: 2
Case #11: NOT POSSIBLE
Case #12: NOT POSSIBLE
Case #13: 4
Case #14: NOT POSSIBLE
Case #15: 4
Case #16: 4
Case #17: NOT POSSIBLE
Case #18: 6
Case #19: 4
Case #20: 7
Case #21: 7
Case #22: 5
Case #23: 5
Case #24: NOT POSSIBLE
Case #25: 1
Case #26: NOT POSSIBLE
Case #27: 2
Case #28: 3
Case #29: 2
Case #30: 3
Case #31: NOT POSSIBLE
Case #32: 1
Case #33: 3
Case #34: 2
Case #35: 3
Case #36: 6
Case #37: 5
Case #38: 2
Case #39: NOT POSSIBLE
Case #40: 1
Case #41: NOT POSSIBLE
Case #42: 5
Case #43: 2
Case #44: NOT POSSIBLE
Case #45: 4
Case #46: NOT POSSIBLE
Case #47: 0
Case #48: 2
Case #49: 3
Case #50: 5
Case #51: 2
Case #52: 5
Case #53: 1
Case #54: 5
Case #55: 2
Case #56: 2
Case #57: NOT POSSIBLE
Case #58: NOT POSSIBLE
Case #59: 0
Case #60: 4
Case #61: 2
Case #62: 5
Case #63: 8
Case #64: 6
Case #65: 1
Case #66: 3
Case #67: 1
Case #68: 3
Case #69: NOT POSSIBLE
Case #70: 7
Case #71: 2
Case #72: NOT POSSIBLE
Case #73: 3
Case #74: 5
Case #75: 5
Case #76: 2
Case #77: 2
Case #78: NOT POSSIBLE
Case #79: 6
Case #80: 6
Case #81: 2
Case #82: 3
Case #83: 5
Case #84: 6
Case #85: 2
Case #86: NOT POSSIBLE
Case #87: 4
Case #88: 6
Case #89: 2
Case #90: 2
Case #91: 3
Case #92: 2
Case #93: 9
Case #94: NOT POSSIBLE
Case #95: NOT POSSIBLE
Case #96: 2
Case #97: 4
Case #98: 1
Case #99: 1
Case #100: 4
